api/queries/user/threadConnection.js

```diff
@@ -16,7 +17,7 @@ export default (
     first,
     after,
     kind,
-  }: { ...PaginationOptions, kind: 'creator' | 'participant' },
+  }: { ...PaginationOptions, kind: 'creator' | 'participant' | 'favorites' },
```

Comment:
We should call this `likes` (or maybe `liked`?) instead of `favorites` to be consistent with the rest of the nomenclature.

Reply:
yes that makes sense, wasn't really sure ( The current UI text is 'Liked') so I'll go with that unless you prefer 'Likes'

Reply:
`liked` works for me!